themes/ohcoe/static/js/scripts.js

```diff
@@ -68,4 +69,30 @@ document.addEventListener('DOMContentLoaded', function() {
     if (scoreContainer && domainScoreGrowth) {
         scoreContainer.innerHTML = 'You have this much growth: <span id="score-container">' + domainScoreGrowth + '%</span>';
     }
+
+    /* Demographic Questions */
+    document.querySelectorAll('.demographic-questions input[type="text"]').forEach(function(el){
+        el.addEventListener('focus', function(e){
+            var radioInput = e.target.parentNode.querySelector('input[type="radio"]');
+            radioInput.setAttribute('checked', 'true');
+        })
+        el.addEventListener('blur', function(e){
+            var radioInput = e.target.parentNode.querySelector('input[type="radio"]');
+            radioInput.setAttribute('value', e.target.value);
+        })
+    });
+    document.querySelector('#demographic-info').addEventListener('submit', function(e){
+        e.preventDefault();
```

Comment:
Any particular reason to query for this demographic information? Is the experience tuned based on the audience?

Reply:
Nope, it was requested by the clients.  The term 'demographic' is used by them, which initially raised my concern, but the demographics here are related to their role and speciality as dentists. Its not asking about categories that one would normally think of as demographic.

As part of the back and forth with the client, we asked that there be a 'Prefer not to answer' option, which is set as the default choice.